doc/classes/ProjectSettings.xml

```diff
@@ -2861,6 +2861,9 @@
 			[b]Note:[/b] Some platforms may restrict the actual value.
 		</member>
 		<member name="rendering/rendering_device/vulkan/max_descriptors_per_pool" type="int" setter="" getter="" default="64">
+			The number of descriptors per pool. Godot's Vulkan backend uses linear pools for descriptors that will be created and destroyed within a single frame. Instead of destroying every single descriptor every frame, they all can be destroyed at once by resetting the pool they belong to.
```

Comment:
```suggestion
			The number of descriptors per pool. The Vulkan rendering driver uses linear pools for descriptors that will be created and destroyed within a single frame. Instead of destroying every single descriptor every frame, they all can be destroyed at once by resetting the pool they belong to.
```
Try to avoid ambiguous "backend", prefer explicit "renderer/rendering method" or "rendering driver", see https://github.com/godotengine/godot/pull/98744.